charts/rasa-x/templates/rasa-x-config-files-configmap.yaml

```diff
@@ -8,8 +8,8 @@ data:
   environments: |
     rasa:
       production:
-        url: "http://{{ include "rasa-x.fullname" . }}-{{ .Values.rasa.versions.rasaProduction.serviceName }}:{{ .Values.rasa.port }}"
+        url: "{{ default "HTTP" .Values.rasa.scheme }}://{{ include "rasa-x.fullname" . }}-{{ .Values.rasa.versions.rasaProduction.serviceName }}:{{ .Values.rasa.port }}"
```

Comment:
Why here don't you use `my-svc.my-namespace.svc` name?

Reply:
my bad!